airflow/api_fastapi/core_api/openapi/v1-generated.yaml

```diff
@@ -8566,6 +8582,40 @@ components:
       - dag_id
       title: DagTagResponse
       description: DAG Tag serializer for responses.
+    DagVersionResponse:
+      properties:
+        id:
+          type: string
+          format: uuid
+          title: Id
+        version_number:
+          type: integer
+          title: Version Number
+        dag_id:
+          type: string
+          title: Dag Id
+        bundle_name:
+          type: string
+          title: Bundle Name
+        bundle_version:
+          anyOf:
+          - type: string
+          - type: 'null'
```

Comment:
Is this default?

Reply:
Yes `null` is the default value (optional). Otherwise it's a string.